.github/workflows/release.yml

```
@@ -1,12 +1,8 @@
 name: Release packages and images
-
 on:
   workflow_dispatch:
   release:
-    types:
-      - published
-      - edited
-
+    types: [published, edited]
```

(unrelated to this PR) could `edited` cause `npm publish` to fail if the release already exists? Or is there a flag such as `--skip-existing`?

---

It looks like `npm publish` doesn't have such a flag, and it's literally [impossible](https://docs.npmjs.com/cli/v7/commands/npm-unpublish#description) to publish the same version number twice for security reasons. Maybe we can just skip this particular job on `edited` events.

---

Something like this https://github.com/iterative/cml/pull/498/files#r626898591?

---

I think the question to ask is... why run at all on `release.edited`?

Only the release notes or attached artefacts would be edited, and neither are used in this workflow.

```suggestion
    types: [published]
```

---

Of course, it's [a good old friend](https://xyproblem.info)